binderhub/static/index.css

```diff
@@ -236,18 +236,27 @@ h4 {
 }
 
 .point-orange {
-    border-color: rgb(243, 162, 83);
-    color: rgb(243, 162, 83);
+    border-color: rgb(247, 144, 42);
+    color: rgb(247, 144, 42);
 }
 
 .point-red {
-    border-color: rgb(208, 102, 129);
-    color: rgb(208, 102, 129);
+    border-color: rgb(204, 67, 101);
+    color: rgb(204, 67, 101);
 }
 
 .point-blue {
-    border-color: rgb(87, 154, 202);
-    color: rgb(87, 154, 202);
+    border-color: rgb(41, 124, 184);
+    color: rgb(41, 124, 184);
```

Comment:
Are you able to share a screenshot of the page with these changes? If not, I can help with that.

Reply:
I will try to share. If I have any problem with it, I will ask for help.

Thanks for the review